java-client/src/main/java/org/opensearch/client/opensearch/core/search/SearchResult.java

```diff
@@ -0,0 +1,639 @@
+/*
```

Comment:
Super nit, but we could only keep this part of the header, this is new file and is not inherited from Elasticsearch